.travis.yml

```diff
@@ -62,6 +62,7 @@ script:
           coverage run --branch --source tlsfuzzer -m unittest discover;
       fi
   - coverage report -m
+  - python verify-scripts-json.py
```

Comment:
I'd add a comment line documenting what is this supposed to do, so someone seeing a failure here can understand what he's supposed to fix